src/test/java/SeamRemoverTest.java

```diff
@@ -0,0 +1,87 @@
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+
+class SeamRemoverTest {
+
+    @Test
+    void removeVerticalSeam() {
+        //given
+        Pixel[][] image = new Pixel[3][5];
+        Pixel pixel00 = mock(Pixel.class);
+        doReturn(1.0).when(pixel00).getCellEnergy();
+        image[0][0] = pixel00;
```

Comment:
You can do this a little simpler by first defining all the `pixelxx` variables and then creating the 2D array with those variables.
```
Pixel[][] image = new Pixel[][] {
{ pixel00, pixel01, ...},
{ pixel10, pixel11, ...}
}
```